guides/doc-Release_Notes/topics/katello-4.11.0.adoc

```diff
@@ -0,0 +1,144 @@
+= Katello 4.11.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1732[Redmine]
+
+== Katello
+
+* pass:[Notification via email when repo sync fail] - https://projects.theforeman.org/issues/17748[#17748]
+
+=== API
+
+* pass:[Remove deprecated docker_tags_whitelist] - https://projects.theforeman.org/issues/36695[#36695]
+* pass:[Remove katello-agent API endpoints] - https://projects.theforeman.org/issues/36665[#36665]
+
+=== Activation Key
+
+* pass:[Add system purpose card to new AK details page] - https://projects.theforeman.org/issues/36610[#36610]
+
+=== Ansible Collections
+
+* pass:[Cannot sync ansible collections repositories to smart proxy] - https://projects.theforeman.org/issues/35746[#35746]
+
+=== Container
+
+* pass:[Taxonomy filtration on Container Image Tags page does not work as expected] - https://projects.theforeman.org/issues/36911[#36911]
+* pass:[ Cannot discovery container repositories on private registries or on registries that only support api v2] - https://projects.theforeman.org/issues/36861[#36861]
+* pass:[Container registries for Sat and Capsule set wrong token expiration field] - https://projects.theforeman.org/issues/36827[#36827]
```

Comment:
```suggestion
* pass:[Container registries for Sat and Capsule set wrong token expiration field] - https://projects.theforeman.org/issues/36827[#36827]
```

Please use either "Foreman Server" and "Smart Proxy Server", or "{ProjectServer}" and "{SmartProxyServer}" if possible.